id/proguard-rules.pro

```diff
@@ -21,58 +21,6 @@
     static void checkParameterIsNotNull(java.lang.Object, java.lang.String);
 }
 
--keep @com.simprints.core.annotations.Unobfuscate public class * {*;}
-
-# Platform calls Class.forName on types which do not exist on Android to determine platform.
--dontnote retrofit2.Platform
-# Platform used when running on Java 8 VMs. Will not be used at runtime.
--dontwarn retrofit2.Platform$Java8
-# Retain generic type information for use by reflection by converters and adapters.
--keepattributes Signature
-# Retain declared checked exceptions for use by a Proxy instance.
--keepattributes Exceptions
-
-#okhttp3
--dontwarn okhttp3.**
--dontwarn okio.**
--dontwarn javax.annotation.**
--dontwarn org.conscrypt.**
-# A resource is loaded with a relative path so the package of this class must be preserved.
--keepnames class okhttp3.internal.publicsuffix.PublicSuffixDatabase
-
-# Gson uses generic type information stored in a class file when working with fields. Proguard
-# removes such information by default, so configure it to keep all of it.
--keepattributes Signature
-
--keepclassmembers enum * { *; }
-
-# These contain serialised models - TODO: should be more selective?
--keep class com.simprints.libsimprints.** { *; }
--keep class com.simprints.infra.eventsync.event.remote.models.** { *; }
--keep class com.simprints.infra.events.event.domain.models.** { *; }
--keep class com.simprints.infra.events.event.local.** { *; }
-
-# Deobfuscations for Crashlytics:
-# https://firebase.google.com/docs/crashlytics/get-deobfuscated-reports
--keepattributes SourceFile,LineNumberTable,*Annotation*
--keep public class * extends java.lang.Exception
--keep public class * extends java.lang.RuntimeException
--keep public class * extends java.lang.Throwable
--keep class com.crashlytics.** { *; }
--dontwarn com.crashlytics.**
-
-# Keep firebase classes. more details at https://github.com/firebase/firebase-android-sdk/issues/2124#issuecomment-1091962221
```

Comment:
Shouldn't we keep (!) these keeps?

Reply:
Issues in comments have been fixed couple of years ago, so it should be safe. Did not cause any issues in my limited testing.